File: src/api/java/RealExpr.java
/**
Copyright (c) 2012-2014 Microsoft Corporation
   
Module Name:

    RealExpr.java

Abstract:

Author:

    @author Christoph Wintersteiger (cwinter) 2012-03-15

Notes:
    
**/ 

package com.microsoft.z3;

/**
 * Real expressions
 **/
public class RealExpr extends ArithExpr
{
    /**
     * Constructor for RealExpr
     **/
    RealExpr(Context ctx, long obj)
    {
        super(ctx, obj);
    }
}

File: src/api/dotnet/RealExpr.cs
/*++
Copyright (<c>) 2012 Microsoft Corporation

Module Name:

    RealExpr.cs

Abstract:

    Z3 Managed API: Real Expressions

Author:

    Christoph Wintersteiger (cwinter) 2012-11-23

Notes:
    
--*/
using System.Diagnostics;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace Microsoft.Z3
{
    /// <summary>
    /// Real expressions
    /// </summary>
    public class RealExpr : ArithExpr
    {
        #region Internal
        /// <summary> Constructor for RealExpr </summary>
        internal RealExpr(Context ctx, IntPtr obj)
            : base(ctx, obj)
        {
            Debug.Assert(ctx != null);
        }
        #endregion
    }
}
